sentry_sdk/integrations/sanic.py

```diff
@@ -32,17 +27,33 @@
 
     from sentry_sdk._types import Event, EventProcessor, Hint
 
+try:
+    from sanic import Sanic, __version__ as VERSION
```

Comment:
```suggestion
    from sanic import Sanic, __version__ as SANIC_VERSION
```

For consistency only.